oauth2client/service_account.py

```diff
@@ -108,6 +112,29 @@ def serialization_data(self):
             'private_key': self._private_key_pkcs8_text
         }
 
+    def to_json(self):
+        strip = (['_private_key'] + self.NON_SERIALIZED_MEMBERS)
```

Comment:
PS I realize this is a bit of a re-design but fits in the "leave it better than we found it" mindset